tests/PackageServiceProviderTests/PackageAssetsTest.php

```diff
@@ -1,25 +1,23 @@
 <?php
 
-namespace Spatie\LaravelPackageTools\Tests\PackageServiceProviderTests;
-
 use Spatie\LaravelPackageTools\Package;
+use function PHPUnit\Framework\assertFileExists;
 
-class PackageAssetsTest extends PackageServiceProviderTestCase
-{
+trait ConfigurePackageAssetsTest {
     public function configurePackage(Package $package)
     {
         $package
             ->name('laravel-package-tools')
             ->hasAssets();
     }
+}
 
-    /** @test */
-    public function it_can_publish_the_assets()
-    {
-        $this
-            ->artisan('vendor:publish --tag=package-tools-assets')
-            ->assertExitCode(0);
+uses(ConfigurePackageAssetsTest::class);
 
-        $this->assertFileExists(public_path('vendor/package-tools/dummy.js'));
-    }
-}
+test('it can publish the assets', function () {
```

Comment:
Please use the `it` functions and remove `if` from the test name

Reply:
Sure, I will update the other PRs as well.